roles/identity-management/manage-rh-sso-identities/templates/manage-users.json.j2

```diff
@@ -0,0 +1,9 @@
+{
+    "username": "{{ sso_user.username | mandatory }}",
```

Comment:
This is where we would want to add the filter to ensure that `username` is lowercase

Reply:
Ok. I will explore this more in detail if we should actually implement this. The reason for such is that, right now, if we specify something in upper case, the user gets automatically created using lowercase characters as the expected behavior from the API, but if we try to run the `delete-user` task, it will not find a user if the value of the variable contains an upper case character.
I think that this is another great example to capture in the README file as you pointed out previously.